WarWithin/PrioryOfTheSacredFlame/Locales/frFR.lua

```diff
@@ -1,22 +1,22 @@
 local L = BigWigs:NewBossLocale("Priory of the Sacred Flame Trash", "frFR")
 if not L then return end
 if L then
-	--L.sacred_flame = "Sacred Flame"
-	--L.guard_captain_suleyman = "Guard Captain Suleyman"
-	--L.forge_master_damian = "Forge Master Damian"
-	--L.high_priest_aemya = "High Priest Aemya"
-	--L.sergeant_shaynemail = "Sergeant Shaynemail"
-	--L.elaena_emberlanz = "Elaena Emberlanz"
-	--L.taener_duelmal = "Taener Duelmal"
-	--L.arathi_knight = "Arathi Knight"
-	--L.arathi_footman = "Arathi Footman"
-	--L.devout_priest = "Devout Priest"
-	--L.fanatical_mage = "Fanatical Mage"
-	--L.lightspawn = "Lightspawn"
+	L.sacred_flame = "Flamme sacrée"
+	L.guard_captain_suleyman = "Capitaine de la garde Suleyman"
+	L.forge_master_damian = "Maître de la forge Damian"
+	L.high_priest_aemya = "Grande prêtresse Aemya"
+	L.sergeant_shaynemail = "Sergent Maÿ"
+	L.elaena_emberlanz = "Elaena Braiselanz"
+	L.taener_duelmal = "Taener Duelmal"
+	L.arathi_knight = "Chevalier arathi"
+	L.arathi_footman = "Fantassin arathi"
+	L.devout_priest = "Prêtre fervent"
+	L.fanatical_mage = "Conjuratrice fanatique"
+	L.lightspawn = "Rejeton de Lumière"
 	L.ardent_paladin = "Paladin ardent"
-	--L.risen_mage = "Risen Mage"
-	--L.sir_braunpyke = "Sir Braunpyke"
+	L.risen_mage = "Mage ressucité"
+	L.sir_braunpyke = "Baron Braunpique"
 
-	--L.baron_braunpyke_warmup_trigger = "They've served their purpose. Baron, demonstrate your worth."
-	--L.custom_on_autotalk_desc = "|cFFFF0000Requires Priest or Paladin.|r Automatically select the NPC dialog option that grants you the 'Blessing of the Sacred Flame' aura."
+	L.baron_braunpyke_warmup_trigger = "Leur mission est remplie. Baron, montrez votre valeur."
+	L.custom_on_autotalk_desc = "|cFFFF0000Requires Priest or Paladin.|r Sélectionne automatiquement le dialogue avec le PNJ qui octroie à votre groupe l'amélioration 'Bénédiction de la Flamme sacrée'."
```

Comment:
should `Requires Priest or Paladin.` be translated too?

Reply:
Thanks, both of your requests have been translated. Some lines have gone through the net, sorry :)